deploy.sh

```diff
@@ -25,11 +25,11 @@ push_helm_chats() {
 }
 
 if [ "${TRAVIS_BRANCH}" == 'csi-v0.3' ]; then
-	export RBD_IMAGE_VERSION='v0.3.0'
-	export CEPHFS_IMAGE_VERSION='v0.3.0'
-elif [ "${TRAVIS_BRANCH}" == 'csi-v1.0' ]; then
-	export RBD_IMAGE_VERSION='v1.0.0'
-	export CEPHFS_IMAGE_VERSION='v1.0.0'
+	export RBD_IMAGE_VERSION='v0.3.0-canary'
+	export CEPHFS_IMAGE_VERSION='v0.3.0-canary'
+elif [ "${TRAVIS_BRANCH}" == 'master' ]; then
+	export RBD_IMAGE_VERSION='v1.0.0-canary'
+	export CEPHFS_IMAGE_VERSION='v1.0.0-canary'
```

Comment:
(Question) What happens to Helm charts, do these get canary versions as well automatically when they are pushed?

Reply:
helm chats will be pushed  only if there is a change in  [version](https://github.com/ceph/ceph-csi/blob/csi-v1.0/deploy.sh#L18)

Reply:
Tried joining the dots, but was unable to, so here are my questions,

- Assume we do 'canary' builds for the plugins, and one of the PRs changed the version in the helm charts, how will this be published [here](https://github.com/ceph/csi-charts), for use with the appropriate plugin version?

- When we make a release, i.e move a version out of 'canary', we would need to push the helm charts for that release as well, even if there are no changes, right?

- As a user, how do I know which helm chart brings in what release of the plugins into my cluster?

Reply:
> Tried joining the dots, but was unable to, so here are my questions,
> 
> * Assume we do 'canary' builds for the plugins, and one of the PRs changed the version in the helm charts, how will this be published [here](https://github.com/ceph/csi-charts), for use with the appropriate plugin version?

this is not there yet to figure out which helm version is support for ce[h-csi version, need to figure it out.
> * When we make a release, i.e move a version out of 'canary', we would need to push the helm charts for that release as well, even if there are no changes, right?
if we are including the helm chats in  the release, yes we have to

> * As a user, how do I know which helm chart brings in what release of the plugins into my cluster?
need to add a document section  for this  one, i will create  an issue